### DISCUSSION TITLE
Webhook response body [Nov 14]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4452] [Discord Threads]
> 🙋‍♂️ **Dr_MHQ**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Refer details here. Anything more required, do ping here   
#### https://docs.nocodb.com/developer-resources/webhooks#accessing-data-handlebars  
#  
> 🙋‍♂️ **Dr_MHQ**  
#### Thanx for the reply … this tutorial explains how to access nocodb data from the Webhooks server… my aim is to access the response I am getting from the Webhooks on nocodb server… for example response body or response status message…  
#  
> 👨‍💻 **pranavxc**  
#### Currently, there is no such option. You can raise a feature request with your requirement.  
#  
> 👨‍💻 **pranavxc**  
#### But we are logging the details in `nc_hook_logs_v2` meta table , it includes failure details. Currently the logs we are not showing in UI  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1038817177773559859)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
